src/metadatacache/MetadataCache.ts

```diff
@@ -93,49 +86,22 @@ export class MetadataCache {
    * @param  {SearchQuery} query Query to filter the DDOs.
    * @return {Promise<QueryResult>}
    */
-  public async queryMetadata(query: SearchQuery): Promise<QueryResult> {
-    const result: QueryResult = await this.fetch
-      .post(`${this.url}${apiPath}/query`, JSON.stringify(query))
+  public async queryMetadata(query: SearchQuery): Promise<any> {
```

Comment:
We really don't know what data structure Aquarius will return on queries anymore?

Reply:
it's elasticseach result. not sure if we should do anything regarding it. That function is more or less useless anyway, people will just directly call aquarius.